larpandoracontent/LArHelpers/LArMvaHelper.h

```diff
@@ -1,26 +1,57 @@
 /**
- *  @file   larpandoracontent/include/LArHelpers/LArSvmHelper.h
+ *  @file   larpandoracontent/include/LArHelpers/LArMvaHelper.h
  *
- *  @brief  Header file for the lar svm helper class.
+ *  @brief  Header file for the lar mva helper class.
  *
  *  $Log: $
  */
-#ifndef LAR_SVM_HELPER_H
-#define LAR_SVM_HELPER_H 1
-
-#include "larpandoracontent/LArObjects/LArSupportVectorMachine.h"
+#ifndef LAR_MVA_HELPER_H
+#define LAR_MVA_HELPER_H 1
 
 #include <fstream>
 #include <chrono>
 #include <ctime>
 
+class AdaBoostDecisionTree;
+class SupportVectorMachine;
+
 namespace lar_content
 {
 
+typedef std::vector<double> DoubleVector; 
+
+/**
+ *  @brief  MvaFeatureTool class template
+ */
+template <typename ...Ts>
+class MvaFeatureTool : public pandora::AlgorithmTool
+{
+public:
+    typedef std::vector<MvaFeatureTool<Ts...> *> FeatureToolVector;
+
+    /**
+     *  @brief  Default constructor.
+     */
+    MvaFeatureTool() = default;
+
+    /**
+     *  @brief  Run the algorithm tool
+     *
+     *  @param  featureVector the vector of features to append
+     *  @param  args arguments to pass to the tool
+     */
+    virtual void Run(DoubleVector &featureVector, Ts... args) = 0;
+};
+
+template <typename ...Ts>
+using MvaFeatureToolVector = std::vector<MvaFeatureTool<Ts...> *>;
```

Comment:
Need some thought about controlling typedefs, as per above (although equivalent to this has been in `LArSupportVectorMachine` for a while).